nixos/modules/misc/nixpkgs.nix

```diff
@@ -97,6 +99,8 @@ in
   };
 
   config = {
-    _module.args.pkgs = import ../../.. config.nixpkgs;
+    _module.args.pkgs = _pkgs;
+    # DEPRECATED
+    _module.args.pkgs-i686 = _pkgs.pkgs-i686;
```

Comment:
Most people don't read the code, they copy working examples.  I think this deserve a proper `builtins.trace` message when used, to at least warn the user that uses of this argument are deprecated.

I will also note that using a new name for a deprecated feature is weird. Rename it to `pkgs_i686`.